Library/Formula/chapel.rb

```diff
@@ -27,8 +27,9 @@ def install
     prefix.install_metafiles
 
     # Install chpl and other binaries (e.g. chpldoc) into bin/ as exec scripts.
-    bin.install Dir[libexec/"bin/darwin/*"]
-    bin.env_script_all_files libexec/"bin/darwin/", :CHPL_HOME => libexec
+    platform = OS::NAME << (OS.linux? ? Hardware::CPU.bits.to_s : "")
```

Comment:
Furthermore, I'm aware that `CPU.bits` is hardcoded to 64 at the moment, but I'm hoping that if in the future it becomes dependent on architecture that this will "just work" on 32-bit.